neural_coder/backends/pytorch_inc_dynamic_quant.yaml

```diff
@@ -17,30 +17,29 @@ transformation:
     - insert_below_model_definition_line
   content:
     - |-
-      [+] if "GraphModule" not in str(type(MODEL_NAME)):
-      [+]     def eval_func(model):
-      [+]         EVAL_FUNC_LINES
-      [+]     try:
-      [+]         torch.backends.quantized.engine = 'onednn'
-      [+]     except:
-      [+]         from torch.backends.quantized import engine; engine = 'onednn'
-      [+]     from neural_compressor.conf.config import QuantConf
-      [+]     from neural_compressor.experimental import Quantization, common
-      [+]     quant_config = QuantConf()
-      [+]     quant_config.usr_cfg.quantization.approach = "post_training_dynamic_quant"
-      [+]     quant_config.usr_cfg.model.framework = "pytorch"
-      [+]     quantizer = Quantization(quant_config)
-      [+]     quantizer.model = common.Model(MODEL_NAME)
-      [+]     quantizer.eval_func = eval_func
-      [+]     MODEL_NAME = quantizer()
-      [+]     MODEL_NAME = MODEL_NAME.model
-      [+]     MODEL_NAME.eval()
-      [+]     try:
-      [+]         with torch.no_grad():
-      [+]             MODEL_NAME = torch.jit.script(MODEL_NAME)
-      [+]         MODEL_NAME = torch.jit.freeze(MODEL_NAME)
-      [+]     except:
-      [+]         pass
+      [+] def eval_func(model):
+      [+]     EVAL_FUNC_LINES
+      [+] try:
+      [+]     torch.backends.quantized.engine = 'onednn'
+      [+] except:
+      [+]     from torch.backends.quantized import engine; engine = 'onednn'
+      [+] from neural_compressor.conf.config import QuantConf
+      [+] from neural_compressor.experimental import Quantization, common
+      [+] quant_config = QuantConf()
```

Comment:
Seems it's still using 1.x API, when will Neural Coder migrate into 2.0 API?

Reply:
I don't know about the high-level plan, from technical perspective, if INC 2.0 API is ready, then migration is doable at any time